NiFiDeploy.groovy

```diff
@@ -10,11 +10,13 @@ import static groovyx.net.http.ContentType.JSON
 import static groovyx.net.http.ContentType.URLENC
 import static groovyx.net.http.Method.POST
 
-
+import java.security.KeyStore
+import org.apache.http.conn.scheme.Scheme
+import org.apache.http.conn.ssl.SSLSocketFactory;
 
 @Grab(group='org.codehaus.groovy.modules.http-builder',
       module='http-builder',
-      version='0.7.1')
+      version='0.7.2')
```

Comment:
@simonellistonball any reason for this upgrade required for this PR or was it just a general pull up?